### PR TITLE
Utilize decltype for slightly improved syntax

### PR DESCRIPTION
**Pull request by Saalvage**

### CODE REVIEW

File: include/assimp/Bitmap.h [hunk 1]
```diff
@@ -75,13 +75,12 @@ class ASSIMP_API Bitmap {
...
-            sizeof(uint32_t) + // size
-            sizeof(uint16_t) + // reserved1
-            sizeof(uint16_t) + // reserved2
-            sizeof(uint32_t);  // offset
+        static constexpr std::size_t header_size =
+            sizeof(decltype(type)) +
+            sizeof(decltype(size)) +
+            sizeof(decltype(reserved1)) +
+            sizeof(decltype(reserved2)) +
+            sizeof(decltype(offset));
```
Comment (JC3) [inline include/assimp/Bitmap.h:83]: You shouldn't need `decltype` here; this is equivalent:

            sizeof(type) +
            sizeof(size) +
            sizeof(reserved1) +
            sizeof(reserved2) +
            sizeof(offset);

File: include/assimp/Bitmap.h [hunk 2]
```diff
@@ -98,22 +97,21 @@ class ASSIMP_API Bitmap {
...
+            sizeof(decltype(width)) +
+            sizeof(decltype(height)) +
+            sizeof(decltype(planes)) +
+            sizeof(decltype(bits_per_pixel)) +
+            sizeof(decltype(compression)) +
+            sizeof(decltype(image_size)) +
+            sizeof(decltype(x_resolution)) +
+            sizeof(decltype(y_resolution)) +
+            sizeof(decltype(nb_colors)) +
+            sizeof(decltype(nb_important_colors));
```
Comment (JC3) [inline include/assimp/Bitmap.h:111]: Same deal here

    sizeof(size) + 
    sizeof(width) +
    ... etc.

**Review by JC3 — CHANGES_REQUESTED** [review]
`sizeof(decltype(expression))` and `sizeof(expression)` are generally equivalent.

File: include/assimp/Bitmap.h (hunk 1 above)
Comment (Saalvage) [reply include/assimp/Bitmap.h]: Yep, I totally missed that somehow, thank you!

**Review by kimkulling — APPROVED** [review]

**Comment by kimkulling** [comment]
Merged, thanks a lot for your contribution.

**Pull request merged**